packages/playground/src/utils/load_deployment.ts

```diff
@@ -61,8 +66,14 @@ export async function loadVms(grid: GridClient, options: LoadVMsOptions = {}) {
     items: data,
   };
 }
+export default function getWireguardConfig(grid: GridClient, name: string) {
```

Comment:
```suggestion
export function getWireguardConfig(grid: GridClient, name: string) {
```

don't export getWireguardConfig as default please